ci/asv.sh

```diff
@@ -9,13 +9,11 @@ RET=0
 if [ "$ASV" ]; then
     echo "Check for failed asv benchmarks"
 
-    pip install git+https://github.com/spacetelescope/asv
-
     cd asv_bench
 
     asv machine --yes
 
-    time asv run --quick | grep "failed"
+    time asv run --quick
```

Comment:
Not directly related to this change (rather to the previous PR that introduced this), but shouldn't this be `asv dev` ? Because I thought `asv run` would create a new env, while everything already has been installed on travis?

Reply:
The link below mentions about `asv dev`

```
This runs a benchmark suite in a mode that is useful during development. It is
equivalent to ``asv run --quick --show-stderr --python=same``
```
So the command should be essentially the same `asv dev`, but I can still change it to `asv dev` nonetheless.


Ref: https://asv.readthedocs.io/en/latest/commands.html#asv-dev

Reply:
There is an essential difference between what you quote (`asv run --quick --show-stderr --python=same`) and what is above, namely the `--python=same`.

And I also think `--show-stderr` is useful for on travis, to be able to easier see what is failing.